bundles/org.eclipse.swt/Eclipse SWT/common/org/eclipse/swt/widgets/Event.java

```diff
@@ -334,15 +334,17 @@ public Rectangle getBounds () {
 	return new Rectangle (x, y, width, height);
 }
 Rectangle getBoundsInPixels () {
-	return DPIUtil.autoScaleUp(getBounds());
+	int deviceZoom = getDeviceZoom();
+	return DPIUtil.autoScaleUp(getBounds(), deviceZoom);
```

Comment:
You can simply inline the method call instead of assigning it a variable (as the method call is self-explaining) like you do in all the other classes.

Reply:
Reverted changes for Event, since it is in common. @akoch-yatta has a separate PR for this.